### DISCUSSION TITLE
Stride 4.2 Release Date

### DISCUSSION THREAD

**Discussion by IndieDev99** [stride3d/stride#2041] [General]
Can anyone tell about the expected Stride 4.2 Release Date. The future of Stride ? Any upcoming features ?

**Comment by Eideren** [comment]
4.2 will release very soon, most of it is ready, we just have a couple of issues to iron out. We'll release a blog post once that's in.

**Reply by IndieDev99** [reply]
Hello! 
December is there. 
Waiting for Stride 4.2

**Reply by Doprez** [reply]
Stride 4.2 is available there are 2 issues that are waiting to be fixed before making the release announcement. 

- The Stride shader extension for VS2022 is broken hopefully fixed soon https://github.com/stride3d/stride/pull/2064
- and the In editor code editor is not working but if you use VS/Rider/Code it wont effect you.
